svgnative/CMakeLists.txt

```diff
@@ -297,9 +330,46 @@ endif()
 ##############################
 # Creating pkg-config file
 ##############################
+include(GNUInstallDirs)
 
-# TODO: Add includes and libraries for individual ports.
 set(BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}")
 set(SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
-set(PRIVATE_LIBS)
-CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib.pc.in" "${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib.pc" @ONLY)
+set(CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
+set(CMAKE_INSTALL_BINDIR "${CMAKE_INSTALL_BINDIR}")
+set(CMAKE_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}")
+set(CMAKE_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_INCLUDEDIR}")
+set(PRIVATE_LIBS "${PRIVATE_LIBS}")
+set(REQUIRES "${REQUIRES}")
+set(PRIVATE_REQUIRES "${PRIVATE_REQUIRES}")
+set(PORTS_INCLUDES "${PORTS_INCLUDES}")
+set(HEADER_SUBDIR "svgnative")
+CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib.pc.in"
+               "${CMAKE_CURRENT_BINARY_DIR}/SVGNativeViewerLib.pc" @ONLY)
```

Comment:
What about naming the file simply `svgnative.pc`?

Reply:
The origin of the naming "SVGNativeViewerLib.pc" is the original patch by Dirk.
But this question is exactly what I wrote as "which is better name, "SVGNativeViewerLib or svgnative" :-). In my understanding, "SVG Native" is the name of a specification of restricted SVG. I cannot exclude a future possibility of other libraries related with SVG Native.

Myself has no special preference on this, I want to hear the comments by @dirkschulze.

Reply:
I don't have a preference either. Can be the simpler name as well.